ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/management-eapi-vrfs.yml

```
@@ -0,0 +1,31 @@
+---
+type: l3leaf
+
+l3leaf:
+  defaults:
+    loopback_ipv4_pool: 10.10.10.0/24
+    vtep_loopback_ipv4_pool: 10.10.10.0/24
+    bgp_as: 65042
+  nodes:
+    - name: management-eapi-vrfs
+      loopback_ipv4_address: 10.42.42.42
+      id: 12
+
+management_eapi:
+  vrfs:
+    - name: VRF_Enable
```

Maybe use this convention as we tend to put input variables in upper case in our tests (I know not always ... :) )

```suggestion
    - name: VRF-ENABLED
```